tests/framework/results/test_context.py

```diff
@@ -250,6 +250,7 @@ def test__get_stratifications(
             ["house"],
             1.53277,
         ),
+        ("wizard_count", "house=='Durmstrang'", None, len, ["house", "familiar"], 4),
```

Comment:
This test passes, but it doesn't make any sense. There aren't 4 member of Durmstrang in each stratification group. There are 0, but `gather_results()` returns an empty dict, which means that any assertion will pass. Our test should be that the returned dict is empty.